Makefile

```diff
@@ -94,7 +94,7 @@ frandom: $(BUILD_DIR)/frandom
 
 fixed: $(BUILD_DIR)/fixed
 
-macapp: set-version
+macapp: avara
```

Comment:
Won't this just build everything twice? Once with our make target, and again with xcodebuild?

Reply:
hmm, good point/catch... it wasn't doing the build-link so that dependency fixed that and did the build but I think you're right that it builds twice